### ISSUE TITLE
Set InfluxDBClient verify_ssl parameter via config file

### ISSUE THREAD

**Issue by HermannSt** [influxdata/influxdb-client-python#142]
I am currently using the verify_ssl parameter that was implemented in the current master branch by the feature request from: https://github.com/influxdata/influxdb-client-python/issues/134. 

__Proposal:__
A verify_ssl option should be added to the InfluxDbClient.from_config_file method.

__Current behavior:__
The verify_ssl option in the config file is ignored which results in an error further down the line when an self signed certificate is used.

__Desired behavior:__
InfluxDbClient.from_config_file recognizes the verify_ssl option and sets the parameter accordingly

__Alternatives considered:__
I have tried the following code with no success:
```
client = InfluxDBClient.from_config_file("config.ini")
client.api_client.configuration.verify_ssl = False
```

__Use case:__
InfluxDBCLient.from_config_file is not usable with a self signed certificate

**Comment by bednar** [comment]
Hi @HermannSt,

thanks for using our client. 

Nice catch, I will fix it.

Regards

**Comment by bednar** [comment]
I've prepared PR https://github.com/influxdata/influxdb-client-python/pull/143.

**Comment by bednar** [comment]
Hi @HermannSt  ,

The issue is fixed in [1.10.0 milestone](https://github.com/influxdata/influxdb-client-python/milestone/12). 

If you would like to use a dev version then install client via:

```pip install git+https://github.com/influxdata/influxdb-client-python.git@master```

Regards
